### ISSUE TITLE
Output raw mail for use with other tools

### ISSUE THREAD

**Issue by Reisen** [charmbracelet/pop#95]
**Is your feature request related to a problem? Please describe.**
Pop seems to be limited at the moment to sending mail via SMTP or Resend, but access to the raw mail body would make this tool work with other email suites and fit better with re-usable command-line philosophy. For example:

- Pipe the output to `msmtp` which can manage many different SMTP targets including local delivery:
  `pop -o - | msmtp -a gmail`
- Pipe the output to `mdeliver` from the mblaze toolkit for managing local delivery targets.
  `pop -o - | mdelivery /var/mail/INBOX`
- Pipe the output through other command line tools to modify mail before sending, I.E `sed`
  `pop -o - | anonymize.sh | tee /tmp/mail.txt | msmtp -a gmail`

Basically right now `pop` is a beautiful UI for creating mail, but does not fit well into the command-line philosophy of being a combine-able tool with the rest of the mail ecosystem.

**Describe the solution you'd like**
A flag for simply outputting the raw resulting mail body/headers without processing. I.E: `pop -o -`

**Describe alternatives you've considered**
Setting up a local mail server that can be targeted by SMTP and scripting that would work but it is a lot of overhead.
